ocaml/xapi/xapi_http.ml

```diff
@@ -44,6 +44,18 @@ open Http
 
 exception NoAuth
 
+let task_id_of_req (req: Http.Request.t) =
+  let all = Http.Request.(req.cookie @ req.query) in
+  if List.mem_assoc "task_id" all
+  then Some (Ref.of_string (List.assoc "task_id" all))
+  else None
+
+let subtask_of_req (req: Http.Request.t) =
```

Comment:
Thanks.@thomassa
Agree with you, I think `let ref_param_of_req req param_name` is better one, it will reduce some duplication and is fit for other `params `in `req`.
I will change it. thanks.